app/src/main/java/de/tum/in/tumcampusapp/component/ui/news/KinoDetailsFragment.java

```diff
@@ -39,7 +41,7 @@
     private String url; // link to homepage
     private LayoutInflater inflater;
 
-    private boolean isBooked = false;
+    private boolean isBooked = true;
```

Comment:
Why would a default to true make sense here?

Reply:
I just want to test the function of the button to show my activity "show_ticket". If "isBooked = false".The button will call the function" buy tickets" , but not " show ticket"

Reply:
Right, I see.